src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -198,7 +198,7 @@ static const string s_output_dir = "cpu_codegen";
 class StaticInitializers
 {
 public:
-    StaticInitializers() { ngraph::file_util::remove_directory(s_output_dir); }
+    StaticInitializers(string directory) { ngraph::file_util::remove_directory(directory); }
```

Comment:
@jbobba @rkimballn1  eventually, we would probably want a global configuration object accessible to all compilations and codegens?